### ISSUE TITLE
Website video disappeared

### ISSUE THREAD

**Issue by Jamesinvi** [CodingTrain/Rainbow-Code#424]
the 69.4 video on genetic algorithms and steering behaviours is no longer available on the website

**Comment by nilsweb** [comment]
Thanks for noticing! This is due to the changes introduced in 19701e5f0e3226008e2e7763654e49704fe8e3ee.

Instead of
``` yaml
contributions:
- title: "JamesInvi's Multiple Organism Simulator (with graph)"
  author:JamesInvi
    name: "James (Invi) Karran"
    url: "https://github.com/Jamesinvi/Animosity"
  url: "https://github.com/Jamesinvi/Animosity"
  source: "https://github.com/Jamesinvi/Animosity"
```
it should be
``` yaml
contributions:
  - title: "JamesInvi's Multiple Organism Simulator (with graph)"
    author:
      name: "James (Invi) Karran"
      url: "https://github.com/Jamesinvi/Animosity"
    url: "https://github.com/Jamesinvi/Animosity"
    source: "https://github.com/Jamesinvi/Animosity"
```
If you like, you can go ahead an fix it.

**Comment by Jamesinvi** [comment]
thank you! I did the file change and pull request. Sorry about this i-m not
familiar with this kind of file

2018-02-10 13:01 GMT+01:00 Nils Weber <notifications@github.com>:

**Comment by nilsweb** [comment]
Don't worry, it's just that one video which isn't available right now. Everything else is fine. And since we haven't established an automated testing suite for this, nobody's to be blamed if something like this slips through.

**Comment by shiffman** [comment]
If anyone is to blame it's me for not checking! But seriously, thank you for the contribution! This website is meant to be a learning experience for all so if things never broke that itself would be a failure!

**Comment by nilsweb** [comment]
Thanks to @Jamesinvi, the video is visible again! Closed via #425

**Comment by shiffman** [comment]
Yay!
